golang/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/google/uuid"
+	"github.com/momentohq/client-sdk-go/momento"
+)
+
+func main() {
+	var AuthToken = os.Getenv("MOMENTO_AUTH_TOKEN")
+	const (
+		CacheName             = "momentocache"
+		ItemDefaultTtlSeconds = 60
+	)
+
+	// Initializes Momento
+	client, err := momento.SimpleCacheClient(&momento.SimpleCacheClientRequest{
+		AuthToken:         AuthToken,
+		DefaultTtlSeconds: ItemDefaultTtlSeconds,
+	})
+	if err != nil {
+		fmt.Println(err.Error())
```

Comment:
would probably just panic in example for now in errors so super obvious to user its failing. the way this is written now it will continue on executing and try to use client that got failed to set up